includes/Special/SpecialNotifications.php

```diff
@@ -41,13 +41,20 @@ public function execute($subpage) {
 		$this->requireLogin();
 
 		$twig = TwiggyWiring::init();
-		$template = $twig->load('@Reverb/special_notifications.twig');
+		$template = $twig->load($this->getContext()->getSkin()->getSkinName() === 'fandomdesktop'
+			? '@Reverb/special_notifications_fandomdesktop.twig'
+			: '@Reverb/special_notifications.twig');
 
 		$groups = self::getNotificationsGroupedByPreference($this->getUser());
 
 		// Additional Scrips for the Notification Page
+		if ($this->output->getContext()->getSkin()->getSkinName() === 'fandomdesktop') {
```

Comment:
You can access the skin directly from `output`: `$this->output->getSkin()->getSkinName()`